edx_proctoring/templates/proctoring/seq_proctored_practice_exam_footer.html

```diff
@@ -0,0 +1,10 @@
+{% load i18n %}
+<div class="faq-proctoring-exam">
+    <p>
+      <a class="footer-link" href="{{link_urls.about_proctored_exams}}" target="_blank">
```

Comment:
please use the link_urls.faq here that configuration has the URL that we need. Thanks